geoengine_swisstopo/i18n/geoengine_swisstopo.pot

```diff
@@ -0,0 +1,85 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* geoengine_swisstopo
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```

Comment:
13.0 ?

Reply:
This is because I did not have to regenerate translations.
Not sure if it should be done.